django_comments/migrations/0001_initial.py

```diff
@@ -8,7 +8,6 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('sites', '0001_initial'),
```

Comment:
If you're creating a migration for a model that uses another model that has no migrations, shouldn't you be able to do that without requiring a '0001_initial' from that other object (since it doesn't exist)?  However, even if it ran without that migration dependency, the 'sites' would still need to be installed for that ForeignKey to be created (forgot about that).

Okay, I guess I'll have to re-think this a bit.  The "placeholder" seems like a good idea but if someone ever turned sites on/off after installing this package they'd likely get some confusing errors.